primitives/core/src/lib.rs

```diff
@@ -83,6 +83,7 @@ mod constants {
 
 	pub const LITENTRY_PARA_ID: u32 = 2013;
 	pub const ROCOCO_PARA_ID: u32 = 2106; // will be replaced by paseo
+	pub const PASEO_PARA_ID: u32 = 2106; // will be replaced by paseo
```

Comment:
"2106" will remain same? So once paseo is merged into dev, rococo needs to be removed immediately, right?

Reply:
I'm not aware about the strategy for removing rococo, So I can't really comment on that.